unitTests/ui/DynamicFacetManagerTest.ts

```diff
@@ -109,6 +109,14 @@ export function DynamicFacetManagerTest() {
       expect(managerContainerChildren().length).toBe(0);
     });
 
+    it('should retrieve the previously removed facets', () => {
```

Comment:
I am not clear on how this test is different from that on line 96.

On line 114, you clear the facets.
On line 115, you load response facets.

How are we checking "previously removed facets" vs. what we received in the most recent response?

Reply:
We check that clearing the facets and they reloading them works. Before the cleared facets couldnt be retrieved.